hawc/static/css/epa-hawc.css

```diff
@@ -1,8 +1,18 @@
 /* Global */
-[href]:focus {
+:focus {
     outline: none !important;
 }
 
+/* disable this rule; breaks svg.d3 layout */
+* {
```

Comment:
removing this css completely adds some jankiness to the logo and button animations @shapiromatron; i'll add it back for those classes if that sounds good to you